engine/class_modules/apl/rogue/outlaw.simc

```diff
@@ -123,7 +124,7 @@ actions.stealth+=/ambush,if=talent.hidden_opportunity
 actions.stealth_cds=variable,name=vanish_opportunity_condition,value=talent.fan_the_hammer.rank+talent.quick_draw+talent.audacity<talent.count_the_odds+talent.keep_it_rolling
 # Hidden Opportunity builds without Crackshot use Vanish if Audacity is not active and when under max Opportunity stacks
 actions.stealth_cds+=/vanish,if=talent.hidden_opportunity&!talent.crackshot&!buff.audacity.up&(variable.vanish_opportunity_condition|buff.opportunity.stack<buff.opportunity.max_stack)&variable.ambush_condition
-# Crackshot builds or builds without Hidden Opportunity use Vanish at finish condition and hold for ADR if they have UHUH
-actions.stealth_cds+=/vanish,if=(!talent.hidden_opportunity|talent.crackshot)&variable.finish_condition&(!talent.underhanded_upper_hand|buff.adrenaline_rush.up)
+# Crackshot builds or builds without Hidden Opportunity use Vanish at finish condition and hold for ADR if they have Underhanded Upperhand + Without a Trace
```

Comment:
Yeah I've not explored Vanish for Double Jeopardy but there is surely something to be found for non-Crackshot builds. I guess since the Tails coin is so strong it could be used to further Tails stacks that we already have instead of just forcing Tails to appear from nothing. Perhaps we could sit on Vanish if you only have a Heads coin unless we hit max charges.